src/components/RefinementList/RefinementList.js

```diff
@@ -146,65 +144,102 @@ export class RawRefinementList extends Component {
 
   render() {
     // Adding `-lvl0` classes
-    const cssClassList = [this.props.cssClasses.list];
-    if (this.props.cssClasses.depth) {
-      cssClassList.push(`${this.props.cssClasses.depth}${this.props.depth}`);
-    }
+    const cssClassList = cx(this.props.cssClasses.list, {
+      [`${this.props.cssClasses.depth}${this.props.depth}`]: this.props
+        .cssClasses.depth,
+    });
+
+    const showMoreButtonClassName =
+      this.props.showMore === true && this.props.canToggleShowMore
+        ? this.props.cssClasses.showMore
+        : cx(
+            this.props.cssClasses.showMore,
+            this.props.cssClasses.disabledShowMore
+          );
 
-    const showMoreBtn =
-      this.props.showMore === true && this.props.canToggleShowMore ? (
+    const showMoreButton =
+      this.props.showMore === true ? (
         <Template
-          rootProps={{ onClick: this.props.toggleShowMore }}
+          rootTagName="button"
```

Comment:
The template is now the button ([see updated story](https://github.com/algolia/instantsearch.js/pull/3152/files#diff-0dd628979285787b60dd3f656a07a619L41)).

Let me know if this is not what we want.